src/app/modules/group/components/group-manager-add/group-manager-add.component.ts

```diff
@@ -59,9 +54,16 @@ export class GroupManagerAddComponent {
         this.login = '';
         this.added.emit();
       },
-      error: () => {
+      error: error => {
         this.state = 'error';
-        this.actionFeedbackService.error($localize`Unable to add this manager.`);
+
+        if (errorIsHTTPNotFound(error)) {
+          this.actionFeedbackService.error($localize`The login you entered does not exist or is not visible to you.`);
+        } else if (errorIsHTTPInternalServer(error)) {
+          this.actionFeedbackService.unexpectedError();
+        } else {
+          this.actionFeedbackService.error($localize`Unable to add this manager.`);
+        }
```

Comment:
I think an `unexpectedError` might happen in other cases that http internal server errors like simple JS errors.
To me here the error mapping is more like:
1. Http Not Found: The login does not exist
2. Http forbidden: Unable to add this manager
3. Any other case: Unexpected error

Reply:
Fixed